javascript/lastfm.js

```diff
@@ -213,8 +213,8 @@ async function fetchPlayData() {
         albumArt.src = "https://lastfm.freetls.fastly.net/i/u/4128a6eb29f94943c9d206c08e625904.jpg";
       }
   } catch (error) {
-    somethingWentWrong("Something went wrong when fetching data from Last.fm. Please try again later.");
-    console.error('Error fetching data:', error);
+      somethingWentWrong(error.message);
```

Comment:
The user will see the raw error message and will understand nothing that it is happening. Merge this line by doing this:

```javascript
somethingWentWrong(`Something went wrong when fetching data from Last.fm. Please try again later or report this error to the developers.\n\n${error.message}`);
```